dg-spark/src/main/code/org/finra/datagenerator/ScalaDataConsumer.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2014 DataGenerator Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.finra.datagenerator
+
+import org.finra.datagenerator.consumer.{DataPipe, DataConsumer}
+import java.util
+
+import scala.collection.JavaConversions._
```

Comment:
This will be flagged as a wildcard import by your scalastyle. I think the wildcard import rule should be disabled, but however you do it for this module, make sure there are no style warnings.